### PR TITLE
Extracting more functions for reuse out of kube-up

### PR DESCRIPTION
**Pull request by davidsiefert**
Needed these bits extracted for reuse.  Makes the script a little cleaner (the kube-up is shorter, easier to read)

### CODE REVIEW

**Comment by davidsiefert** [comment]
Signed CLA

**Comment by wojtek-t** [comment]
@fgrzadkowski - Filip it's somewhat related to what you're trying to achieve also. Can you please review?

File: cluster/aws/util.sh [hunk 1]
```diff
@@ -1016,7 +1029,9 @@ function start-minions() {
       --tags ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=Name,Value=${NODE_INSTANCE_PREFIX} \
              ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=Role,Value=${MINION_TAG} \
              ResourceId=${ASG_NAME},ResourceType=auto-scaling-group,Key=KubernetesCluster,Value=${CLUSTER_ID}
+}
 
+function wait-minions {
```
Comment (fgrzadkowski) [inline cluster/aws/util.sh]: Why do you split this into two functions?

**Comment by fgrzadkowski** [comment]
In general lgtm. Just one question inline.

File: cluster/aws/util.sh (hunk 1 above)
Comment (davidsiefert) [inline cluster/aws/util.sh]: Underlying reason was I'm using these scripts to recreate the auto-scaling group and the minion counting doesn't add up when creating a new auto-scaling group (1 existing minion + 1 new minion = 2 minions > 1 expected minion).  By separating this I'm able to skip the check in my own scripting.  The additional benefit of extracting the function is shorter function bodies ~ cleaner code.

**Comment by fgrzadkowski** [comment]
LGTM

**Pull request merged**